package.json

```diff
@@ -31,7 +31,7 @@
     "package-all": "yarn run package -- --all",
     "rebuild-and-e2e": "yarn run build && yarn run test-e2e",
     "start": "cross-env NODE_ENV=production electron ./",
-    "start-headless": "cross-env NODE_ENV=production electron ./dist/headless-bundle.js",
+    "start-headless": "cross-env NODE_ENV=production node ./dist/headless-bundle.js",
```

Comment:
This is going to cause a lot of pain, because it'll require to rebuild native modules against node.